sw/device/tests/BUILD

```diff
@@ -3461,6 +3461,25 @@ opentitan_test(
     ],
 )
 
+opentitan_test(
+    name = "usbdev_aon_pullup_test",
+    srcs = ["usbdev_aon_pullup_test.c"],
+    exec_env = {
+        "//hw/top_earlgrey:fpga_cw310_test_rom": None,
```

Comment:
With a real host, this test probably makes the OS complain a bit--We aren't responding to the usual device configuration sequence, right? Does the line check work reliably in that environment?

Reply:
A number of these very simple SiVal tests are indeed not very friendly to the host in that they ignore traffic, but because Endpoint Zero has not been configured, we'll just be receiving a few SOF packets and SETUP packets that we ignore.

It's quite unlikely that we'd sample more than a few '0' bits on the line when it's expected to be '1' and line_check() has some resilience against that, as well as sampling for 2ms (= 2 bus frames). The test passes on the CW310 FPGA.